go/adbc/driver/flightsql/flightsql_adbc.go

```diff
@@ -742,39 +973,16 @@ func (c *cnxn) SetOption(key, value string) error {
 
 	switch key {
 	case OptionTimeoutFetch:
```

Comment:
any reason to use `fallthrough` as opposed to just having the multiple cases?

`case OptionTimeoutFetch, OptionTimeoutQuery, OptionTimeoutUpdate:` ?

Reply:
i can update, not a big deal